### ISSUE TITLE
New work

### ISSUE THREAD

**Issue by timathom** [yale-datachemist/entity-resolution#60]
* Updated the (README)[https://github.com/yale-datachemist/entity-resolution/tree/main/data/test] for the benchmark data. 
* Was able to add the new generated IDs for each name.

On the server:
* The ops file for the complete dataset is located in `/data/ops/marc_ops_combined.json`.
* The benchmark data is indexed in `data/vector_storage`, domain `benchmark`, commit `b1`.

**Comment by matko** [comment]
>  Also, when calling vectorlink scan-neighbors for an intra-domain comparison, both A--B and B--A are compared, but we only need A--B because the relationship is undirected/intransitive.
Are you sure?
scan-neighbors only opens and searches in one index. if A is your indexed domain, and B is your sequence domain, Only A's index is opened, and we search for all elements in B using the A index. So the search is one direction.

**Comment by timathom** [comment]
Right, but I was using the same domain as both A and B to try to cluster within the domain :)

**Comment by matko** [comment]
apologies, i misunderstood!
For an intra-compare, there might already a better approach in vectorlink in the form of the `duplicates` subcommand. This does a nearest neighbors search for each element in the index. We could explore if that's an option here.
